src/libraries/System.Data.Odbc/.editorconfig

```diff
@@ -0,0 +1 @@
+dotnet_diagnostic.IDE0060.severity = none
```

Comment:
Why special-case this one lib?  Is that just temporary because you haven't gotten to it yet?

Reply:
Actually, that's IDE0060 not IDE0059.  Why is that in this PR?

Reply:
Just mistake in Draft PR